### PR TITLE
Add support for ZWave Aeotec ZW099 dimmer module

### PR DESCRIPTION
**Pull request by dhylands**

### CODE REVIEW

File: zwave-classifier.js [hunk 1]
```diff
@@ -23,17 +23,27 @@ const COMMAND_CLASS_METER = 50;               // 0x32
...
+    // able to support this by adding a read of current each time we get a
+    // power changed.
     zwInfo: {
-      manufacturerId: '0x0086',
+      manufacturerId: AEOTEC_MANUFACTURER_ID,
     },
     excludeProperties: ['current'],
   },
   {
+    // The Aeotech ZW096 says it supports the MULTILEVEL command class, but
```
Comment (mrstegeman) [inline zwave-classifier.js]: Aeotec, not Aeotech

File: zwave-classifier.js [hunk 2]
```diff
@@ -23,17 +23,27 @@ const COMMAND_CLASS_METER = 50;               // 0x32
...
+    // setting it acts like a no-op. So we remove the level property this so
```
Comment (mrstegeman) [inline zwave-classifier.js]: `level property so`

File: zwave-classifier.js [hunk 3]
```diff
@@ -23,17 +23,27 @@ const COMMAND_CLASS_METER = 50;               // 0x32
...
 
+const AEOTEC_MANUFACTURER_ID = '0x0086';
+const AEOTEC_ZW096_PRODUCT_ID = '0x0060';
+
 const QUIRKS = [
   {
+    // The Aeotec devices don't seem to notify on current changes, only on
+    // instantaneous power changes. So we exclude this for now. We might be
+    // able to support this by adding a read of current each time we get a
+    // power changed.
```
Comment (mrstegeman) [inline zwave-classifier.js]: 'change'

File: zwave-classifier.js (hunk 1 above)
Comment (dhylands) [reply zwave-classifier.js]: For some reason my fingers almost always an extra character when I type Aeotec. Usually its a t :)

**Comment by dhylands** [comment]
I addressed the comments so far.

**Pull request merged**